remotespark/__init__.py

```diff
@@ -1,2 +1 @@
-from .RemoteSparkMagics import *
```

Comment:
Can you explain what this line was doing? I ran into problems from nosetests because of it.